git-coverage

```diff
@@ -399,6 +399,9 @@ class GccCoverage:
                 if not match:
                     continue
                 gcov = match.group(1)
+                if gcov != "*.gcov":
```

Comment:
This is a string match against *.gcov; this shouldn't work.
It needs to be:
if gcov != "%s.gcov"%filename:
    os.unlink(gcov)
    continue